core/src/es/adapter/transformer/strategy/initialized.rs

```diff
@@ -0,0 +1,47 @@
+//! [`Initialized`] [`Strategy`] definition.
+
+use std::marker::PhantomData;
+
+use futures::{stream, TryStreamExt as _};
+
+use crate::es::event;
+
+use super::{AsIs, Strategy};
+
+/// [`Strategy`] for wrapping [`Event`]s in [`Initial`].
+///
+/// [`Event`]: crate::es::Event
+/// [`Initial`]: event::Initial
+#[derive(Clone, Debug)]
+pub struct Initialized<InnerStrategy = AsIs>(PhantomData<InnerStrategy>);
```

Comment:
@ilslv this shouldn't exist. `Initial` has meaning only when it's about applying `Event` to some `State`. It has no meaning when we're talking about `Event`s only, which is the `event::Adapter` case.